nengo/dists.py

```diff
@@ -189,20 +189,29 @@ class UniformHypersphere(Distribution):
 
     Parameters
     ----------
+    min_magnitude : Number
+        The closed lower bound of the uniform hypersphere distribution;
+        such that the sampled vector magnitudes >= min_magnitude.
+        The default value and minimum value for min_magnitude is 0.
+        The maximum value for min_magnitude is 1. Values outside this range
+        will be clipped to this range.
     surface : bool
         Whether sample points should be distributed uniformly
         over the surface of the hyperphere (True),
         or within the hypersphere (False).
+        Note: Setting surface to True generates sample points from the
+        hypersphere surface regardless of what the value of min_magnitude is.
```

Comment:
I'd remove this note, and in the description of `min_magnitude` say that it's ignored if `surface == True`.